doc/rules-resources.md

```diff
@@ -143,9 +143,7 @@ Macro to orchestrate an objc_library with generated sources for intentdefiniton
 | <a id="objc_intent_library-name"></a>name |  <p align="center"> - </p>   |  none |
 | <a id="objc_intent_library-src"></a>src |  <p align="center"> - </p>   |  none |
 | <a id="objc_intent_library-class_prefix"></a>class_prefix |  <p align="center"> - </p>   |  <code>None</code> |
-| <a id="objc_intent_library-class_visibility"></a>class_visibility |  <p align="center"> - </p>   |  <code>None</code> |
 | <a id="objc_intent_library-testonly"></a>testonly |  <p align="center"> - </p>   |  <code>False</code> |
-| <a id="objc_intent_library-swift_version"></a>swift_version |  <p align="center"> - </p>   |  <code>None</code> |
```

Comment:
_Should_ these two be used? Seems like a bug?

Reply:
This is `objc_library`, I think `class_visibility` and `swift_version` aren't applicable. Not sure why they are added here before. /cc @steeve